### ISSUE TITLE
Terminate task dialog doesn't terminate

### ISSUE THREAD

**Issue by aeschli** [Microsoft/vscode#19267]
Testing #19057
- create a task in task.json
```json
{
    "version": "0.1.0",
    "isShellCommand": true,
    "showOutput": "always",
    "tasks": [
        {
            "taskName": "yeoman",
            "command": "yo"
        } 
    ],
    "_runner": "terminal" 
}
```
- run the task
- while the task is running, run a second task
- dialog asks:
There is an active running task right now. Terminate it first before executing another task.
Also note that the output view is made visible.
- Click on 'Terminate Running Task'
- Second dialog shows up: 
The tasks are executed in the intergrated terminal. Use the terminal to manage the tasks.

Would be better to only show one dialog. Why restrict the tasks so that only one can run at a time? The terminal gives the possibility to manage them individually.

**Comment by dbaeumer** [comment]
@aeschli did you try to start the same task again. The terminal runner support running n task, but only n different once. I thought it is too confusing to be able to start the same task n times. I fully agree that the wording and flow needs to be improved. In terminal it should be:

- detect that the same task is started again. Ask whether the task should be started again, ignored or the current one killed and restarted

Not critical for Jan.

**Comment by dbaeumer** [comment]
See also discussion in  #19243  about focus handling and the output window.

**Comment by aeschli** [comment]
I guess it really depends on the task if it makes sense to run it more than once.
For simplicity I'd suggest that any task, even the same one, can be ran in parallel.

**Comment by dbaeumer** [comment]
For the terminal task runner there is now a picker that shows the active tasks.

**Comment by isidorn** [comment]
@aeschli please verify

**Comment by aeschli** [comment]
Verified that 'Terminate Running Task' now correctly works.

Still, the Output view is brought to front he the question is asked, for no clear reason. @dbaeumer Let me know if you want a bug for this or all will change anyways.

**Comment by dbaeumer** [comment]
@aeschli please open an issue.
